apps/agreement/test/agreement/agreement_new_action.js

```diff
@@ -156,6 +234,7 @@ contract('Agreement', ([_, owner, submitter, someone]) => {
 
         context('when the app is unregistered', () => {
           beforeEach('mark as unregistered', async () => {
+            await disputable.changeSetting({ aragonAppFeesCashierAddress: ZERO_ADDRESS, from: owner })
```

Comment:
Do we actually need this one?

Reply:
Yes, otherwise it tries to pay for fees and it complains about not enough balance.